portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,53 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
```

Comment:
I'd suggest you personalize your title as well. This is what shows in the browser tab.